bus_enhanced/readme.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
File should be exactly `README.rst` in order to be recognised by Linux platforms. Can you please rename it?